thirdparty/faiss/faiss/IndexIVF.cpp

```diff
@@ -1310,14 +1313,15 @@ size_t InvertedListScanner::scan_codes(
     return nup;
 }
 
-void InvertedListScanner::scan_codes_range(
+size_t InvertedListScanner::scan_codes_range(
```

Comment:
Why we have to change the interface, which is too heavy. `res.nres` can get the number of results

Reply:
great !

Reply:
good catch, has changed it back.